packages/flutter_tools/lib/src/asset.dart

```diff
@@ -30,6 +30,9 @@ const String kFontManifestJson = 'FontManifest.json';
 // Should match '2x', '/1x', '1.5x', etc.
 final RegExp _assetVariantDirectoryRegExp = RegExp(r'/?(\d+(\.\d*)?)x$');
 
+// We assume the main asset is designed for a device pixel ratio of 1.0
+const double _defaultResolution = 1.0;
```

Comment:
constants should be prefixed with `k` according to the style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k